### ISSUE TITLE
[BUG] Forge Servers do not work

### ISSUE THREAD

**Issue by SHXRKIEgg** [How-Bout-No/Outvoted#103]
**Describe the bug**
Mod Crashing Server side 1.17.1

**To Reproduce**
1. Add Mods to Server
2. Run Server
3. Crash
4. See mod loading error

**Screenshots/Logs**
https://pastebin.com/eXzhtY6C

**Version (please complete the following information):**
 - Minecraft Version [1.17.1]
 - Modloader [Forge]
 - Mod Version [outvoted-1.5.0-beta.3]

**Additional context**
I even tested standalone by installing on Curseforge and it installs the required mods automatically and used them for server and still crashes

**Comment by How-Bout-No** [comment]
Uh... very weird. Obviously the log blames Outvoted but nothing in the actual crash has anything to do with it from what I can see. I'll see if I can get a basic server running on my end.

**Comment by How-Bout-No** [comment]
Yup, confirmed crashing on my end. Really odd. I have very little to go off of here I'll see what I can do.

**Comment by SHXRKIEgg** [comment]
Yeah was trying to add it to my Better MC Modpack. Thanks the help

**Comment by How-Bout-No** [comment]
Figured I should issue an update: unfortunately, not much progress has been made. This crash is out of my abilities and I'm not getting much update from others who are much more knowledgeable than I am. I will still continue experimenting with my end fwiw.

**Comment by Zegveld** [comment]
The ArrayIndexOutOfBoundsException seems to be caused by `net.minecraft.client.model.HumanoidModel` class not being found.

Don't know why but when it tries to load the `io.github.how_bout_no.outvoted.forge.ModEventBusSubscriber` class it also wants to load `software.bernie.geckolib3.renderers.geo.GeoArmorRenderer` which in its turn causes the attempt to load the `net.minecraft.client.model.HumanoidModel` class.

Hope this helps with narrowing it down. I might have some more time later this week to do some more debugging.
